### PR TITLE
218-reduced-basis-module-document-fails-to-execute-in-jupyter

### PR DESCRIPTION
**Pull request by thomas-saigre**
- [ ] #218
- [x] fix errors due to bad xref or include

### CODE REVIEW

File: docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc [hunk 1]
```diff
@@ -4,112 +4,355 @@ include::user:ROOT:partial$header-macros.adoc[]
...
+import feelpp.mor as mor
 import feelpp
 ----
 
 
+== Set the environment
+
+[source, python]
+----
+config = feelpp.globalRepository("reducedbasis")
```
Comment (prudhomm) [inline docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc:35]: use `localRepository("reducedbasis")` for jupyter

Comment (prudhomm) [reply docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc:35]: or `localRepository("")`

File: docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc [hunk 2]
```diff
@@ -4,112 +4,355 @@ include::user:ROOT:partial$header-macros.adoc[]
...
+
+
 == Set the model
 
-To declare the objects, use these lines :
+Set the parameters nedded to the model :
+- `config_file` : path to the cfg file
+- `dim` : the dimension of the case
+- `order` : the order of discretization (`1` for stem:[\mathbb{P}_1], `2` for stem:[\mathbb{P}_2])
+- `time_dependant` : is the case stationnary or transient ? (for now always to `False`)
```
Comment (prudhomm) [inline docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc:50]: type : dependent (e not a)

File: docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc [hunk 3]
```diff
@@ -4,112 +4,355 @@ include::user:ROOT:partial$header-macros.adoc[]
...
+- `name` : name of the CRB case
 
 [source,python]
 ----
-DIM = 2
-heatBox=heat(dim=DIM,order=1)
-heatBox.init()
-model = toolboxmor(dim=DIM, time_dependent=False)
-model.setFunctionSpaces( Vh=heatBox.spaceTemperature())
+config_file = "thermal-fin.cfg"
```
Comment (prudhomm) [inline docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc:55]: this is not robust. Do you create the file ? or should it be available somewhere ?

File: docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc [hunk 4]
```diff
@@ -4,112 +4,355 @@ include::user:ROOT:partial$header-macros.adoc[]
...
+dim = 2
+order = 1
+time_dependant = False
+name = "thermalfin-2d"
+----
+
+[source,python]
+----
+cfg = feelpp.readCfg(config_file)
```
Comment (prudhomm) [inline docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc:64]: what is this function `readCfg` ?

File: docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc [hunk 5]
```diff
@@ -4,112 +4,355 @@ include::user:ROOT:partial$header-macros.adoc[]
...
+feelpp.Environment.setConfigFile(config_file)
+----
```
Comment (prudhomm) [inline docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc:66]: no expected output provided ?

File: docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc [hunk 6]
```diff
@@ -4,112 +4,355 @@ include::user:ROOT:partial$header-macros.adoc[]
...
+crb_model_properties = mor.CRBModelProperties(worldComm=feelpp.Environment.worldCommPtr())
+crb_model_properties.setup(crb_model_file)
+crb_model_outputs = crb_model_properties.outputs()
+
+output_names = []
+for n, _ in crb_model_outputs:
+    output_names.append(n)
+
+print(f"Outputs of the models are {output_names}")
+----
```
Comment (prudhomm) [inline docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc:99]: no output provided ?

Comment (prudhomm) [reply docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc:99]: what should we expect ?

File: docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc [hunk 7]
```diff
@@ -4,112 +4,355 @@ include::user:ROOT:partial$header-macros.adoc[]
...
+heatBox.init()
+----
+
+The parameter `default_parameter` is defined in the section `Parameters` of the JSON file.
+
+[source, python]
+----
+modelParameters = heatBox.modelProperties().parameters()
+default_parameter = modelParameters.toParameterValues()
+----
```
Comment (prudhomm) [inline docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc:116]: should we expect something ?
print the parameter may be ?

File: docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc [hunk 8]
```diff
@@ -4,112 +4,355 @@ include::user:ROOT:partial$header-macros.adoc[]
...
+
+
+[source, python]
```
Comment (prudhomm) [inline docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc:119]: describe the next block with respect to the math (affine decomposition...)

File: docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc [hunk 9]
```diff
@@ -4,112 +4,355 @@ include::user:ROOT:partial$header-macros.adoc[]
...
+
+[%collapsible.result]
+.Results
+====
+----
+[reducedbasis] Online rb initialized
+[slepc4py] number of (smaller) eigenvalues computed : 16
+[reducedbasis] Constant of coercivity : 0.9999999999999977
+[slepc4py] number of eigenvalues computed : 16
+[reducedbasis] Constant of continuity : 1.0000000000000018
```
Comment (prudhomm) [inline docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc:262]: continuity and coercivity are the same ? does it mean that the parameter for use is the one for the scalar product ?

Comment (prudhomm) [reply docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc:262]: make a note for this

**Review by prudhomm — CHANGES_REQUESTED** [review]
the testcase is never presented I think. 
Recall the equations and the objectives from a mathematical point of view.

**Comment by prudhomm** [comment]
@thomas-saigre I have merged your changes as it fixes things. However could you add some more explanations ?

**Comment by prudhomm** [comment]
@thomas-saigre I didn't delete the branch, you can still work in it

**Comment by thomas-saigre** [comment]
@prudhomm I have pushed some modifications in the branch `saigre/pyfeelpp` that needed to be updated and merged, see #222

**Pull request merged**